VERSION

```diff
@@ -1 +1 @@
-0.5-dev
+0.6.0-dev
```

Comment:
I can alternatively set this to 0.5.1-dev if we expect to follow up with other fixes shortly and don't want to bump the version too quickly